doc/book/zend.barcode.objects.md

```diff
@@ -0,0 +1,403 @@
+|getInstructions()                  |Array    |Return drawing instructions as an array.                                                                               |
+|getHeight($recalculate = false)    |Integer  |Return the height of the barcode calculated after possible rotation                                                    |
+|getWidth($recalculate = false)     |Integer  |Return the width of the barcode calculated after possible rotation                                                     |
+|getOffsetTop($recalculate = false) |Integer  |Return the position of the top of the barcode calculated after possible rotation                                       |
+|getOffsetLeft($recalculate = false)|Integer  |Return the position of the left of the barcode calculated after possible rotation                                      |
+
+
+
+
+## Description of shipped barcodes
+
+You will find below detailed information about all barcode types shipped by default with Zend
+Framework.
+
+### Zend\\Barcode\\Object\\Error
+
+![image](../images/zend.barcode.objects.details.error.png)
+
+This barcode is a special case. It is internally used to automatically render an exception caught by
+the `Zend\Barcode` component.
+
+### Zend\\Barcode\\Object\\Code128
+
+![image](../images/zend.barcode.objects.details.code128.png)
+
+- **Name:** Code 128
+- **Allowed characters:** the complete ASCII-character set
+- **Checksum:** optional (modulo 103)
+- **Length:** variable
+
+There are no particular options for this barcode.
+
+### Zend\\Barcode\\Object\\Codabar
+
+![image](../images/zend.barcode.objects.details.codabar.png)
+
+- **Name:** Codabar (or Code 2 of 7)
+- **Allowed characters:**'0123456789-\$:/.+' with 'ABCD' as start and stop characters
+- **Checksum:** none
+- **Length:** variable
+
+There are no particular options for this barcode.
+
+### Zend\\Barcode\\Object\\Code25
+
+![image](../images/zend.barcode.objects.details.code25.png)
+
+- **Name:** Code 25 (or Code 2 of 5 or Code 25 Industrial)
+- **Allowed characters:**'0123456789'
+- **Checksum:** optional (modulo 10)
+- **Length:** variable
+
+There are no particular options for this barcode.
+
+### Zend\\Barcode\\Object\\Code25interleaved
+
+![image](../images/zend.barcode.objects.details.int25.png)
+
+This barcode extends `Zend\Barcode\Object\Code25` (Code 2 of 5), and has the same particulars and
+options, and adds the following:
+
+- **Name:** Code 2 of 5 Interleaved
+- **Allowed characters:**'0123456789'
+- **Checksum:** optional (modulo 10)
+- **Length:** variable (always even number of characters)
+
+Available options include:
+
+
+|Option        |Data Type|Default Value|Description                                               |
+|--------------|---------|-------------|----------------------------------------------------------|
+|withBearerBars|Boolean  |FALSE        |Draw a thick bar at the top and the bottom of the barcode.|
+
+ .. note:
```

Comment:
Cannot recall: didn't you find a way to convert notes to blockquotes or similar, @GeeH?